Package.swift

```diff
@@ -68,5 +68,5 @@ let package = Package(
     ],
     Target.allTests()
   ] as [[Target]]).flatMap { $0 },
-  swiftLanguageVersions: [.v5]
+  swiftLanguageVersions: [.v5_1]
```

Comment:
for swift 5.1 there is not a new language version (odd... but that's it), so languageVersions can remain v5. I guess that the constraint will come from swift-tools-version.

Reply:
Yup, noticed this yesterday. Unintuitive but I guess can make some sense :)